docs/src/main/asciidoc/security-oidc-configuration-properties-reference.adoc

```diff
@@ -19,14 +19,8 @@ include::{generated-dir}/config/quarkus-oidc.adoc[opts=optional, leveloffset=+1]
 
 * xref:security-oidc-bearer-token-authentication.adoc[OIDC Bearer token authentication]
 * xref:security-oidc-bearer-token-authentication-tutorial.adoc[Protect a service application by using OpenID Connect (OIDC) Bearer token authentication]
-// * https://www.keycloak.org/documentation.html[Keycloak Documentation]
 * https://openid.net/connect/[OpenID Connect]
-// * https://tools.ietf.org/html/rfc7519[JSON Web Token]
 * xref:security-openid-connect-client-reference.adoc[OpenID Connect and OAuth2 Client and Filters Reference Guide]
-// * xref:security-openid-connect-dev-services.adoc[Dev Services for Keycloak]
```

Comment:
shouldn't these not be uncommented rather than removed?

Reply:
Thanks, @maxandersen. 
Based on the following comment from line 32, it seemed like these commented lines were to be eliminated.
```
// TASK - Select some references and eliminate the rest.
```
I'll revert this set of changes. 
Please let me know if you would like me to uncomment them.

Reply:
@rolfedh I don't think they were meant to be eliminated, but only hidden from the product

Reply:
I've reverted the changes.

Reply:
@rolfedh they are still shown as removed, I thought you'd prefer to conditionalise these lines ?